jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/WebStepsUtils.java

```diff
@@ -0,0 +1,63 @@
+package org.jbehavesupport.core.internal.web;
+
+import lombok.RequiredArgsConstructor;
+import org.jbehavesupport.core.web.WebAction;
+import org.jbehavesupport.core.web.WebActionContext;
+import org.jbehavesupport.core.web.WebActionResolver;
+import org.jbehavesupport.core.web.WebProperty;
+import org.jbehavesupport.core.web.WebPropertyContext;
+import org.jbehavesupport.core.web.WebPropertyResolver;
+import org.jbehavesupport.core.web.WebWaitCondition;
+import org.jbehavesupport.core.web.WebWaitConditionContext;
+import org.jbehavesupport.core.web.WebWaitConditionResolver;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.EnableRetry;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@EnableRetry
+@RequiredArgsConstructor
+@Component
+public class WebStepsUtils {
+    public static final String ACTION = "action";
+    public static final String ELEMENT = "element";
+    public static final String PROPERTY = "property";
+
+    private final WebPropertyResolver propertyResolver;
+    private final WebWaitConditionResolver waitConditionResolver;
+    private final WebActionResolver actionResolver;
+
+    @Retryable(
+        value = { StaleElementReferenceException.class },
+        backoff = @Backoff(delay = 1000))
```

Comment:
I think that 1000 delay is default, so you should be able to omit it 🙂

Reply:
It is, thanks